lib/src/services/database.dart

```diff
@@ -13,6 +13,24 @@ class Database extends Service {
     fromFirestore: (snapshot, _) => UserProfile.fromJson(snapshot.data()!),
     toFirestore: (user, _) => user.toJson(),
   );
+
+  /// A collection of [SellerProfile] objects.
+  CollectionReference<SellerProfile> get sellers => firestore.collection("sellers").withConverter<SellerProfile>(
+    fromFirestore: (snapshot, _) => SellerProfile.fromJson(snapshot.data()!),
+    toFirestore: (sellers, _) => sellers.toJson(),
```

Comment:
Just FYI, the `fromFirestore` and `toFirestore` are about individual documents, so this should be in the singular form (seller, not sellers)

Reply:
I fixed this in my own commit, just press Resolve when you've read this